annotation-server/src/medications/medication.entity.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { DrugDto } from './dtos/drugbank.dto';
+import { Medication } from './medication.entity';
+
+describe('Medication Entity', () => {
+    const exampleName = 'Ibuprofen';
+    const exampleDescription =
+        'Ibuprofen is a medication in the nonsteroidal anti-inflammatory drug (NSAID) class that is used for treating pain, fever, and inflammation.[7] This includes painful menstrual periods, migraines, and rheumatoid arthritis.[7] It may also be used to close a patent ductus arteriosus in a premature baby.[7] It can be used by mouth or intravenously.[7] It typically begins working within an hour.';
+    const exampleExternalIdentifierPharmGKB = {
+        resource: 'PharmGKB',
+        identifier: '123456789ABCDEF!',
+    };
+    const exampleExternalIdentifierDailyMed = {
+        resource: 'DailyMed',
+        identifier: 'foo-987654321',
+    };
+    const exampleExternalIdentifierWikipedia = {
+        resource: 'Wikipedia',
+        identifier: 'insalatamista999',
+    };
+    const exampleInternationalBrand1 = 'Advil Migraine';
+    const exampleInternationalBrand2 = 'Medipren';
+
+    describe('during creation', () => {
```

Comment:
changed it to ``it()`` because if the test fails and there is only "describe", the text isn't shown to me (although it is with ``it()``)

Reply:
actually, I would prefer keeping it nested because it makes this really nice structure when running a single test suite :3 also, I think this structure is preferable for future tests
![Screenshot from 2022-05-04 15-57-35](https://user-images.githubusercontent.com/57364771/166697277-7e0a0b41-5a57-46a7-b275-c1f2defe0609.png)
.